dotcom-rendering/src/web/components/Nav/ExpandedMenu/Columns.tsx

```diff
@@ -127,12 +130,82 @@ const searchBar = css`
 	${from.desktop} {
 		display: none;
 	}
-	margin-left: 45px;
+	box-sizing: border-box;
+	display: block;
+	margin-left: 13px;
+	max-width: 23.75rem;
+	position: relative;
 	margin-bottom: 24px;
 	margin-right: 41px;
 	padding-bottom: 15px;
 `;
 
+const searchInput = css`
+	${textSans.large()}
+	background-color: rgba(255,255,255, .1);
+	border: 0;
+	border-radius: 1000px;
+	box-sizing: border-box;
+	color: ${neutral[100]};
+	height: 36px;
+	padding-left: 38px;
+	vertical-align: middle;
+	width: 100%;
+	&::placeholder {
+		color: ${neutral[100]};
+	}
```

Comment:
Does it work if we preserve the existing styles from Source here? I know this ticket is about aligning with Frontend but I think within that goal we can also allow some deviation to preserve the design system standards for text inputs